src/api/app/views/webui/request/_delete_request_dialog.html.haml

```diff
@@ -0,0 +1,22 @@
+.dialog.darkgrey_box
+  .box.box-shadow
+    %h2.box-header Create Delete Request
+    .dialog-content
+      %p
+        Do you really want to request the deletion of #{project_or_package_link project: @project, package: @package}?
+      %p.smaller
+        You are about to send a request to the owner of the original project or package. Please, describe
```

Comment:
We can be specific here. It's either a project or a package. So in the end, it's `owner of the original project` or `owner of the original package`.